src/ngx-drag-scroll.ts

```diff
@@ -685,6 +694,24 @@ export class DragScrollComponent implements OnDestroy, AfterViewInit, OnChanges,
     return scrollLeftPos >= this._contentRef.nativeElement.scrollWidth;
   }
 
+  @HostListener('wheel', ['$event'])
+  private onWheel(event: WheelEvent) {
+    if (this._xWheelEnabled) {
+      event.preventDefault();
+
+      if (this._snapDisabled) {
+          console.log('x');
```

Comment:
Actually I've just noticed that console.log left behind :/
Will you fix that on the nearest occasion or should I make another PR?

Reply:
Ah, I need to pay more attention to code reviews. I'll get it fixed.

Reply:
Great, thanks!